.gitignore

```diff
@@ -7,3 +7,7 @@ npm-debug.log
 
 # Mac OS X
 .DS_Store
+
+!intl/
+intl/*
+!intl/en/
```

Comment:
IMO, if we actually went with it, it should be a combination of both. Categories and alphabetically inside each category. Anyways, enough :bike: ;)